work-here/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
let's use the existing package.json as reference. [this will do](https://github.com/nearform/the-fastify-workshop/blob/master/src/step-12-database/package.json), just change the name to "work-here"

Reply:
https://github.com/nearform/the-fastify-workshop/pull/32/commits/397efc7611b17cc52fd7afb44f3b692b09c65565